lti_tool/templates/lti_tool/_siblings.html

```diff
@@ -0,0 +1,60 @@
+{% load i18n %}
+{% load course %}
+{% load exercise %}
+<nav class="row siblings" aria-label="{% translate 'PAGINATION' %}">
+
+	{% if previous %}
+	{% if previous.submittable %}
+	{% exercise_accessible previous as accessible %}
+	{% else %}
+	{% module_accessible previous as accessible %}
+	{% endif %}
+	{% if accessible and previous.is_revealed or accessible and previous.type == "module" %}
+	<a href="{{ previous.link }}" class="nav-flex-left">
+		<span aria-hidden="true">&laquo;</span> {{ previous.name|parse_localization }}
+	</a>
+	{% else %}
+	<span class="pull-left disabled">
+		<span aria-hidden="true">&laquo;</span> {{ previous.name|parse_localization }}
+		{% if is_course_staff %}
+		<a href="{{ previous.link }}">
+			<span class="glyphicon glyphicon-lock" aria-hidden="true"></span>
+			{% translate "EARLY_ACCESS" %}
+		</a>
+		{% endif %}
+	</span>
+	{% endif %}
+	{% endif %}
+
+	{% if upper_name %}
+	<span class="hidden-md hidden-sm hidden-xs" style="display: block; margin: auto; text-align: center;">
+		<a href="{{ upper_link }}">
+			{{ upper_name|parse_localization }}
+		</a>
+	</span>
+	{% endif %}
```

Comment:
Yes. I was also thinking about this, but then thought to follow the earlier principle of keeping the LTI templates separate, and have no LTI-related modifications to the original templates. But we could also do it like this, to avoid repeated code.

Reply:
Well, this isn't critical, so I guess you will skip this now to save time.